### ISSUE TITLE
SteamProvider and SteamProviderUtils

### ISSUE THREAD

**Issue by N1cE-me** [yesitskev/steamchat#2]
I'm import this project to eclipse but I have only 2 troubles - SteamProvider.java and SteamProviderUtils.java
In other java's this files using like 

```
SteamProvider.Interaction.CONTENT_URI
```

or

```
SteamProviderUtils.UserPersona
```

And show errors.
How fix it?

P.s
Screen with errors
![Screen with errors](http://doc74.ru/sct1.png)

**Comment by yesitskev** [comment]
Yeah the latest commit has a bunch of stuff that doesn't work.  Ill do a cleanup tomorrow so that its at least runnable. 

If i don't get to this, you can ditch the data persistence completely. It has nothing to do with steam itself (swap it out for sqlbrite or something)

**Comment by yesitskev** [comment]
Or to save time, you could go back to commit f84c0a888ba7a27ebeed8496bf8b091bbbe3f1b7

That _SHOULD_ work.
